wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/SuccessfulCertificateEnrollmentViewController.swift

```diff
@@ -115,6 +115,7 @@ final class SuccessfulCertificateEnrollmentViewController: AuthenticationStepVie
         self.isUpdateMode = isUpdateMode
         super.init(nibName: nil, bundle: nil)
         setupViews()
+        notifyE2EICertificateChange()
```

Comment:
is there a better place to send this notification? like when the updateCertificate use case is finished?

Reply:
We don't have "UpdateCertificate use case". We use to unroll and update certificate 
`EnrollE2EICertificateUseCase` because it's essentially the same thing. I moved `notifyE2EICertificateChange()` to the `EnrollE2EICertificateUseCase`